# This patch for /usr/sbin/multistrap is required to
# make "noauth=true" config works properly
# See: https://github.com/volumio/Build/issues/348
--- /usr/sbin/multistrap	2019-02-17 17:04:08.475975948 +0000
+++ multistrap	2019-02-17 17:04:02.215909932 +0000
@@ -317,6 +317,7 @@
 $config_str .= " -o Apt::Architecture=" . shellescape($arch);
 $config_str .= " -o Dir::Etc::TrustedParts=" . shellescape("${dir}${etcdir}trusted.gpg.d");
 $config_str .= " -o Dir::Etc::Trusted=" . shellescape("${dir}${etcdir}trusted.gpg");
+$config_str .= " -o Acquire::AllowInsecureRepositories=true";
 $config_str .= " -o Apt::Get::AllowUnauthenticated=true"
 	if (defined $noauth);
 $config_str .= " -o Apt::Get::Download-Only=true";
